### PR TITLE
Script to filter messages in a queue.

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
It adds a script to filter all messages in a RabbitMQ queue. It is written so that is extensible with other filters.

The included filter **niche-import-same-or-after-date** is only for Niche users stuck in the `userImportQueue`.

This script was used to filter 33K messages in the `userImportQueue` queue in our Quicksilver RabbitMQ instance.

#### How to review
- 👀

### CODE REVIEW

File: filter-messages-from-queue/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - It assumes the messages stored in the queue are JSON objects. **Any messages that are not JSON parseable are rejected and not re-queued**
+    - This filter checks for messages with the property `activity_timestamp` that has a timestamp with a date being same or after the one set in `FILTER_DATE`.
+
+### Message schemas.
+The **niche-import-same-or-after-date** filter assumes messaged being filtered follow this schema.
+```
+{
+  "first_name": "maynard",
+  "last_name": "",
+  "email": "maynard@crownpawnshop.com",
```
Comment (aaronschachter) [inline filter-messages-from-queue/README.md:23]: Just checking.. is this [real data](http://crownpawnshop.com/)?

File: load-tests/README.md [hunk 2]
```diff
@@ -67,72 +67,26 @@ Option | Description | native
 **-s** | *Scenario*. Which load test scenario to run. | `F`
 **-n** | *Amount of phones*. Number of phones we will be using for this broadcast. Starts at `+15551111110`. Max is `+15559111110` (8 Million). | `F`
 **--delay** | *Delay*. Delay the execution of the tests by x amount of seconds on each iteration. Used to add more flexibility when calibrating `rps`. | `F`
-**--ng** | *Not Generator*. Don't use the mobile number generator. Use the env variable set mobile number for all tests | 'F'
+**--ng** | *Not Generator*. Don't use the mobile number generator. Use the env variable set mobile number for all tests | `F`
```
Comment (aaronschachter) [inline load-tests/README.md]: Not expecting to see all this here now that #13 has changed... 🤷‍♂️ ?

**Review by aaronschachter — APPROVED** [review]
Looks good, although might want to change the sample data to be less real looking? (or totes change it if is real)

File: filter-messages-from-queue/README.md (hunk 1 above)
Comment (rapala61) [reply filter-messages-from-queue/README.md:23]: It is... from a 🎥

**Pull request merged**